### PR TITLE
Updating to latest release (0.10.1910.3107)

### PR DESCRIPTION
**Pull request by anpaz-msft**

### CODE REVIEW

**Comment by anpaz-msft** [comment]
Binder checks: https://mybinder.org/v2/gh/Microsoft/QuantumKatas/release/v0.10

File: NuGet.config [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (tcNickolas) [inline NuGet.config]: It doesn't seem like we should need this file for accessing the published NuGets?

File: tutorials/SingleQubitGates/SingleQubitGates.csproj [hunk 2]
```diff
@@ -6,9 +6,9 @@
   </PropertyGroup>
```
Comment (tcNickolas) [inline tutorials/SingleQubitGates/SingleQubitGates.csproj]: Should line 3 here be updated to `netcoreapp3.0` as well? This seems to fail CI (Sorry, GitHub doesn't let me leave a comment there)

File: NuGet.config (hunk 1 above)
Comment (anpaz-msft) [reply NuGet.config]: Not for the released, but yes for the pre-released. We are adding a similar file to all our repos so it's easier to integrate x-repo changes before they become part of a release.

File: tutorials/SingleQubitGates/SingleQubitGates.csproj (hunk 2 above)
Comment (anpaz-msft) [reply tutorials/SingleQubitGates/SingleQubitGates.csproj]: yeap. this project was added after the migration of the rest was done so it was not picked up.
I'll update.

**Pull request merged**